bindata/network/multus/multus.yaml

```diff
@@ -116,29 +117,109 @@ spec:
       tolerations:
       - operator: Exists
       serviceAccountName: multus
-      initContainers:
-      - name: multus-binary-copy
+      containers:
+      - name: kube-multus
         image: {{.MultusImage}}
-        command: ["/entrypoint/cnibincopy.sh"]
+        command: [ "/bin/bash", "-ec", "--" ]
+        args:
+          - >
+            /entrypoint/cnibincopy.sh;
+            exec /entrypoint.sh
+            --multus-conf-file=auto
+            --multus-autoconfig-dir=/host/var/run/multus/cni/net.d
+            --multus-kubeconfig-file-host=/etc/kubernetes/cni/net.d/multus.d/multus.kubeconfig
+{{- if eq .DefaultNetworkType "OpenShiftSDN"}}
+            --readiness-indicator-file=/var/run/multus/cni/net.d/80-openshift-network.conf
+{{- else if eq .DefaultNetworkType "OVNKubernetes"}}
+            --readiness-indicator-file=/var/run/multus/cni/net.d/10-ovn-kubernetes.conf
+{{- end}}
+            --cleanup-config-on-exit=true
+            --namespace-isolation=true
+            --multus-log-level=verbose
+            --cni-version=0.3.1
+            --additional-bin-dir=/opt/multus/bin
+            --skip-multus-binary-copy=true
         resources:
           requests:
             cpu: 10m
-            memory: 10Mi
+            memory: 150Mi
```

Comment:
I forgot to ask, does multus actually use 150mi of memory?

Reply:
Trevor already asked, expected this to change

Reply:
Thanks, apparently I missed this. I set to 65Mi, using your 55 megs as a baseline and then added 15% and rounded up to the nearest 5 to give a little extra.